ice/mvc/model.zep

```diff
@@ -178,13 +184,9 @@ abstract class Model extends Arr implements \Serializable
      */
     public static function findOne(var filters = null, array options = [])
     {
-        var result, model, instance;
-
-        let model = get_called_class(),
-            instance = create_instance(model),
-            result = instance->loadOne(filters, options);
-
-        return result;
+        var model;
+        let model = new static(filters, options);
```

Comment:
Options are not the same as data in construct.

Options are passed to the db::find query
```
// eg. find the latest active user
$user = User::findOne(['status' => 1], ['order' => 'id DESC']);
```

Data is used to populate the fields
```
// fills the $user->data attribute
$user = new User(null, ['username' => 'ice', 'status' => 1]);
```